cisco_secure_endpoint/manifest.json

```diff
@@ -0,0 +1,40 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "9636c2eb-34f6-4aa4-a236-c39e47b21c79",
+    "app_id": "cisco-secure-endpoint",
+    "display_on_public_website": false,
+    "tile": {
+        "overview": "README.md#Overview",
+        "configuration": "README.md#Setup",
+        "support": "README.md#Support",
+        "changelog": "CHANGELOG.md",
+        "description": "Gain insights into Cisco Secure Endpoint logs",
+        "title": "cisco_secure_endpoint",
```

Comment:
`cisco-secure-endpoint`

Reply:
Have made the change.

Reply:
title must be “Cisco Secure Endpoint" @ankitarajput-crest  (cc: @jnhunsberger )